make-release.sh

```diff
@@ -33,55 +76,114 @@ bump_version () {
 
   echo "Updating project version to ${NEXT_VERSION}"
   # change version/version.go file
-  sed -i version/version.go -r -e "s#(Version = \")(v?[0-9.]+)(\")#\1${NEXT_VERSION}\3#g"
+  sed -i version/version.go -r -e "s#(Version = \")(v?[0-9.]+)(\")#\1v${NEXT_VERSION}+rc\3#g"
+  sed -i deploy/templates/components/csv/clusterserviceversion.yaml -r -e "s#(name: devworkspace-operator.)(v[0-9.]+)#\1v${NEXT_VERSION}#g"
+  sed -i deploy/templates/components/csv/clusterserviceversion.yaml -r -e "s#(version: )([0-9.]+)#\1${NEXT_VERSION}#g" 
   if [[ ! -z $(git status -s) ]]; then # dirty
-    git add version/version.go
     COMMIT_MSG="[release] Bump to ${NEXT_VERSION} in ${BUMP_BRANCH}"
     git commit -asm "${COMMIT_MSG}"
   fi
   git pull origin "${BUMP_BRANCH}"
 
   set +e
-  PUSH_TRY="$(git push origin "${BUMP_BRANCH}")"
+  [[ -v "$DRY_RUN" ]] && PUSH_TRY="$(git push origin "${BUMP_BRANCH}")" || PUSH_TRY="protected branch hook declined"
   # shellcheck disable=SC2181
   if [[ $? -gt 0 ]] || [[ $PUSH_TRY == *"protected branch hook declined"* ]]; then
     PR_BRANCH=pr-${BUMP_BRANCH}-to-${NEXT_VERSION}
     # create pull request for the main branch branch, as branch is restricted
     git branch "${PR_BRANCH}"
     git checkout "${PR_BRANCH}"
     git pull origin "${PR_BRANCH}"
-    git push origin "${PR_BRANCH}"
+    [[ -v "$DRY_RUN" ]] && git push origin "${PR_BRANCH}" || echo "Dry run: push ${PR_BRANCH}"
     lastCommitComment="$(git log -1 --pretty=%B)"
-    hub pull-request -f -m "${lastCommitComment}" -b "${BUMP_BRANCH}" -h "${PR_BRANCH}"
+    [[ -v "$DRY_RUN" ]] && hub pull-request -f -m "${lastCommitComment}" -b "${BUMP_BRANCH}" -h "${PR_BRANCH}" || echo "Dry run: create PR"
```

Comment:
Outside of the scope of this PR, but should we move from [`hub`](https://github.com/github/hub) to [`gh`](https://github.com/cli/cli)?